### PR TITLE
Small change to ensure 5'Flank annotations show up, new tests added

### PR DESCRIPTION
**Pull request by alexramos**
Flank annotations are still hardcoded to 5' 5kb and 3' 0kb

### CODE REVIEW

File: test/EnsemblTranscriptDatasourceTest.py [hunk 1]
```diff
@@ -218,6 +218,35 @@ def test_overlapping_gene_5flank(self):
         txs = ds.get_overlapping_transcripts("22", 22224920, 22224920)
         self.assertTrue(len(txs) == 0)
 
+    @TestUtils.requiresDefaultDB()
+    def test_5_prime_flank_annotation_positive_strand(self):
```
Comment (LeeTL1220) [inline test/EnsemblTranscriptDatasourceTest.py]: How about a test that looks at 5' (both + and - transcripts) but at a distance just outside of the flank.  To make sure IGR is caught on an edge case.
